docs/user/aws/limits.md

```diff
@@ -73,12 +169,32 @@ If you intend to start with a higher number of workers, enable autoscaling and l
 or a different instance type, please ensure you have the necessary remaining instance count within the instance type's
 limit to satisfy the need. If not, please ask AWS to increase the limit via a support case.
 
+You can then confirm the limits and usage (for on Demand Instances) by running the following:
+
+```sh
+$ aws support describe-trusted-advisor-check-result --check-id 0Xc6LMYG8P | jq -c '.result.flaggedResources[] | {service_limit: (.metadata[2]), status: (.status), region: (.region), limit: (.metadata[3]), usage: (.metadata[4])}'
+```
+
 ## Elastic Load Balancing (ELB/NLB)
 
 By default, each cluster will create 2 network load balancers for the master API server (1 internal, 1 external) and a
 single classic elastic load balancer for the router. Additional Kubernetes LoadBalancer Service objects will create
 additional [load balancers][load-balancing].
 
+You can then confirm the limits and usage (for on Load Balancers) by running the following:
+
+```sh
+$ aws support describe-trusted-advisor-check-result --check-id iK7OO0l7J9 | jq -c '.result.flaggedResources[] | {service_limit: (.metadata[2]), status: (.status), region: (.region), limit: (.metadata[3]), usage: (.metadata[4])}'
```

Comment:
This (and your following `describe-load-balancers`) are for classic load balancers (used by Ingress).  They do not cover the network load balancers used for the Kubernetes/OpenShift API.  Two ways to get both:

```console
$ aws elbv2 describe-account-limits --query "Limits[? @.Name == 'network-load-balancers'].Max" --output text
500
$ aws elbv2 describe-load-balancers --query 'join(`\n`, @.LoadBalancers[].LoadBalancerArn)' --output text | wc -l
66
$ aws elb describe-account-limits --query "Limits[? @.Name == 'classic-load-balancers'].Max" --output text
80
$ aws support describe-trusted-advisor-check-result --check-id iK7OO0l7J9 --query 'result.flaggedResources[].{service_limit: @.metadata[2], status: @.status, limit: @.metadata[3], usage: @.metadata[4]}' --output text
80	Active load balancers	ok	7
```

Reply:
What should I show here? 
Is `$ aws support describe-trusted-advisor-check-result --check-id iK7OO0l7J9 --query 'result.flaggedResources[].{service_limit: @.metadata[2], status: @.status, limit: @.metadata[3], usage: @.metadata[4]}' --output text` enough?

Reply:
@wking take a look at what I did here (I feel like I am missing something).

Reply:
I think you still (as of 4909f0462ce) need an additional blurb after the Classic Load Balancer request, with content like:

    And for	Network	Load Balancers:

    ```sh
    aws elbv2 describe-account-limits --query "Limits[? @.Name == 'network-load-balancers'].Max" --output text
    aws elbv2 describe-load-balancers --query 'join(`\n`, @.LoadBalancers[].LoadBalancerArn)' --output text | wc -l
    ```

But really, why don't we drop this all into a `scripts/aws-check-limits.sh` or some such so the user just has one thing to call?

Reply:
> But really, why don't we drop this all into a `scripts/aws-check-limits.sh`...

I've submitted this in #1343, in case folks want to see what it would look like.  If it looks good, I suggest we land that PR and then reroll this one to replace the local commands with a single script call.

Reply:
I fixed the calls to show the limits. We still need to discuss #1343 but we can do that on that PR.